tests/testthat/test-quo.R

```diff
@@ -128,6 +128,15 @@ test_that("quosures are spliced", {
   expect_identical(quo_text(q), "foo(bar(baz))")
 })
 
+test_that("quo_text() uses as_string encoding repair (#611)", {
+  old <- Sys.getlocale("LC_CTYPE")
+  on.exit(Sys.setlocale("LC_CTYPE", old))
```

Comment:
Probably best to avoid locale in the test, just create a symbol of a unicode tag.

Reply:
Right. I’ll clean it after 🍝.